.ci/Jenkinsfile

```diff
@@ -144,6 +155,40 @@ def cleanup(){
   unstash 'source'
 }
 
+def generateTestSignWithElasticStage() {
+  return {
+    withNode(labels: "ubuntu-20 && immutable", sleepMax: 20, forceWorkspace: true) {
+      cleanup()
+      dir("${BASE_DIR}"){
+        withMageEnv(){
+          sh(label: 'Install elastic-package',script: "make install")
+          sh(label: 'Prepare for tests (build zipped packages)',script: "make build-unsigned-zip-for-tests")
+          googleStorageUpload(bucket: env.INFRA_SIGNING_BUCKET_ARTIFACTS_PATH,
+            credentialsId: env.INTERNAL_CI_JOB_GCS_CREDENTIALS,
+            pathPrefix: 'build/integrations/',
+            pattern: 'build/integrations/*.zip',
+            sharedPublicly: false,
+            showInline: true)
+          withCredentials([string(credentialsId: env.JOB_SIGNING_CREDENTIALS, variable: 'TOKEN')]) {
+            triggerRemoteJob(auth: CredentialsAuth(credentials: 'local-readonly-api-token'),
+              job: 'https://internal-ci.elastic.co/job/elastic+unified-release+master+sign-artifacts-with-gpg',
+              token: TOKEN,
+              parameters: "gcs_input_path=${env.INFRA_SIGNING_BUCKET_ARTIFACTS_PATH}",
+              useCrumbCache: true,
+              useJobInfoCache: true)
+          }
+          googleStorageDownload(bucketUri: "${env.INFRA_SIGNING_BUCKET_SIGNED_ARTIFACTS_PATH}/*",
```

Comment:
To be ownest with you I have never tried downloading everything in that bucket as it could kill the CI worker :D It's the same job we're using to sign Beats artifacts.